resources/assets/js/videos/components/thumbnailPreview.vue

```diff
@@ -81,10 +99,30 @@ export default {
             let nbrCols = Math.sqrt(this.thumbnailsPerSprite);
             let nbrRows = Math.ceil(nbrThumbnailsOnSprite / nbrCols);
             return [nbrCols, nbrRows];
+        },
+        hoverTimeText() {
+            return Vue.filter('videoTime')(this.hoverTime);
+        },
+        hoverTimeStyle() {
+            return { 'font': `bold ${this.fontSize}px Sans-Serif`, 'color': '#cccccc', 'bgColor': '#222' };
+        },
+        xtext() {
+            return this.canvasWidth / 2;
+        },
+        ytext() {
+            // compute font size in pixel
+            return this.canvasHeight + this.fontSizeInPx + (this.hoverTimeBarHeight - this.fontSizeInPx) / 2;
+        },
+        fontSizeInPx() {
+            return this.fontSize * (72 / 96);
```

Comment:
Is this dependent on the DPI value of the display?

Reply:
I don't know. I didn't use have another display. So it could be. I use this function just to center the time vertically.

Reply:
It looked ok on my screen which is scaled 1.5x. I was just wondering about the numbers. 72 and 96 DPI are common values for screens but phones or other retina displays can have different values.

As this is a constant value, we can probably just skip the computation and take 14.5 * 72 / 96 ~= 11 as `fontSize`.